docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,340 @@
+
+It is important to know that if you create a Prometheus snapshot skipping the WAL file the **last hour of data** will not be present in the snapshot. For example, if the Prometheus logs show the following operations:
+
+```bash
+level=info ts=2018-12-05T11:00:08.289897499Z caller=compact.go:398 component=tsdb msg="write block" mint=1543989600000 maxt=1543996800000 ulid=01CXYSH0M8RSWNJG3YM1S9YGY5
+level=info ts=2018-12-05T13:00:09.242667991Z caller=head.go:446 component=tsdb msg="head GC completed" duration=170.063092ms
+```
+
+A good time to create a snapshot of the TSDB  to perform the migration would be at 13:01 UTC, so you create the snapshot just after Prometheus wrote the latest information stored in the WAL file to disk.
+If you do this, you will find the following:
+
+[Insert image of  Prometheus query with 1 hour of missing data due to snapshot]
+
+You would have lost 1 hour of Prometheus metrics.
+
+In order to ensure a minimal data loss, the new Prometheus deployment should be deployed 55 mins (ensuring with this less than 5 mins of data loss) before the next TSDB compaction happens, so the new Prometheus deployment can start scraping data before the snapshot is restored. Continuing with the last example:
+
+If the next TSDB compaction happens at 13:00 UTC you should deploy BKPR Prometheus in the cluster at 12:05 UTC, so Prometheus can start scraping data and fill the gap that the snapshot will create. Then, at 13:00 UTC you can create the Prometheus snapshot and will restore it in the new deployment, creating the following:
+
+![prometheus_successful_migration](../../images/prometheus-migration/tsdb_restored.png)
+
+## Step 4: Create a TSDB snapshot and restore it in the new deployment
+
+Now that the process has been explained in detail, these are the commands that will have to be executed in order to migrate the TSDB from the old Prometheus deployment to the BKPR one.
+
+After a previous sync with the TSDB, the command to generate the Prometheus snapshot is:
+
+`curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot?skip_head=true`
+
+That command is meant to be executed from the sidecar rsync container that is deployed in the old Prometheus deployment. As soon as you execute that command, the snapshot will be stored under the data directory of Prometheus, under `snapshots/`.
+
+Once the snapshot is done, connect to the sidecar rsync container in the new Prometheus deployment and execute:
+
+`rsync -avrP prometheus-rsync.NAMESPACE.svc.cluster.local::data/snapshots/ /opt/bitnami/prometheus/data/`
+
+> Note:  Make sure you change the NAMESPACE placeholder for the Kubernetes namespace where the old Prometheus deployment lives.
+
+After the rsync has finished, move the block directories to the main data directory of Prometheus. This an example of the structure you will find data directory of the new Prometheus deployment:
```

Comment:
This is an

you will find in the data directory

Reply:
done.